src/js/App/GlobalFilter/GlobalFilter.js

```diff
@@ -84,6 +107,13 @@ const GlobalFilter = () => {
         }
     }, [selectedTags, isAllowed()]);
 
+    useEffect(() => {
+        const sapTag = workloads?.[0]?.tags?.[1];
+        if (sapCount !== undefined && sapTag) {
```

Comment:
I would maybe go for `typeof sapCount === 'number'` instead. This will still allow strings, arrays, etc.

Reply:
Good point.